altdoc/mkdocs_static.yml

```diff
@@ -7,7 +7,8 @@ repo_name: $ALTDOC_PACKAGE_NAME
 
 ### Plugins
 plugins:
-  - search
+  - search:
+      separator: "[\\s\\-]+|(pl_|DataFrame_|LazyFrame_|Expr_|ExprArr_|ExprBin_|ExprCat_|ExprDT_|ExprList_|ExprMeta_|ExprName_|ExprStr_|ExprStruct_)"
```

Comment:
Can't it just be an underscore `_`?

Reply:
Nope, the results are not good with this when we look for functions that have a `_` like `sum_horizontal()` or `concat_str()`. I'll add a section in the preprocessing script to automatically add those prefixes.